CMSgov/easi-app
# [EASI-3124] Not it gov request action

# EASI-3124

[Figma
](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=1-24823&mode=design&t=FtBIpeVhthdEQacD-0)

## Changes and Description

- Not an IT Governance Request action form
- Displays modal if edits have been requested



## How to test this change

- Log in as IT Gov admin with `itGovV2Enabled` flag turned on
- Test for requests both with and without edits requested (modal should show on submit for edits requested):
  - Go to /governance-review-team/:systemIntakeId/actions
  - Select "Issue a decision or close this request"
  - Select "Not an IT Governance Request"
  - Test action form



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
